client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/MongoHeuristicsCalculator.java

```diff
@@ -281,6 +283,50 @@ private double calculateDistanceForInvertedType(InvertedTypeOperation operation,
         return !Objects.equals(actualType, expectedType) ? 0.0 : MIN_DISTANCE_TO_TRUE_VALUE;
     }
 
+    private double calculateDistanceForNearSphere(NearSphereOperation operation, Object doc) {
+        String field = operation.getFieldName();
+        Object actualPoint = getValue(doc, field);
+
+        double x1 = Math.toRadians(operation.getLongitude());
+        double y1 = Math.toRadians(operation.getLatitude());
+        double x2;
+        double y2;
+
+        // GeoJsonPoint in document
+        if (isDocument(actualPoint) && getValue(actualPoint, "type").equals("Point") && getValue(actualPoint, "coordinates") instanceof List<?>) {
```

Comment:
is this case sensitive? ie, always be `Point` and never `point`?

Reply:
It's always Point as it's explained [here](https://datatracker.ietf.org/doc/html/rfc7946#section-1.4).

Reply:
ok. thx. add a link to that as comment

Reply:
Link added